packages/blockchain/src/index.ts

```diff
@@ -401,7 +400,7 @@ export default class Blockchain implements BlockchainInterface {
 
   private async cliqueUpdateVotes(header?: BlockHeader) {
     // Block contains a vote on a new signer
-    if (header && !header.coinbase.toBuffer().equals(CLIQUE_EMPTY_BENEFICIARY)) {
+    if (header && !header.coinbase.isZero()) {
```

Comment:
These kind of changes are really a proof that it makes so much sense to work on these kind of PRs together! 😄

Reply:
(so everyone has his special open-eyes-sections)